package.json

```diff
@@ -117,6 +117,7 @@
 		"@hashicorp/react-button": "^6.3.1",
 		"@hashicorp/react-code-block": "^6.4.0",
 		"@hashicorp/react-consent-manager": "^9.2.0",
+		"@hashicorp/react-design-system-tokens": "^0.0.0-canary.99e8874",
```

Comment:
Can yup update to [0.0.0-nightly.d49b06f](https://www.npmjs.com/package/@hashicorp/react-design-system-tokens/v/0.0.0-nightly.d49b06f)? The npm version is a bit of a fib since we've never published a stable version, so the "latest" is just the first published version.